inc/dbmysql.class.php

```diff
@@ -615,16 +562,13 @@ static function optimize_tables($migration=NULL, $cron=false) {
       return $nb;
    }
 
-
    /**
-    * Get  information about DB connection for showSystemInformations
-    *
-    * @since version 0.84
+    * Get information about DB connection for showSystemInformations
     *
-    * @return Array of label / value
+    * @since 0.84
+    * @return string[] Array of label / value
     */
    public function getInfo() {
-
       // No translation, used in sysinfo
       $ret = array();
       $req = $this->request("SELECT @@sql_mode as mode, @@version AS vers, @@version_comment AS stype");
```

Comment:
Hum... Good catch; I did not pay attention this query miss a `FROM` clause; let as it, dbiterator is not (yet) designed to use functions ;)